influunt-api/conf/application.conf

```diff
@@ -363,7 +363,7 @@ play {
     modules {
         enabled += "be.objectify.deadbolt.java.DeadboltModule"
         enabled += "AppDeadboltHook"
-        enabled += "CentralModule"
+        #enabled += "CentralModule"
```

Comment:
Esse por enquanto vai ficar comentado, quem for usar descomenta. A maioria não utiliza a central